multiautocomplete/src/main/java/com/teamwork/autocomplete/adapter/BaseTypeAdapterDelegate.java

```diff
@@ -51,13 +51,15 @@
 import java.util.regex.Pattern;
 
 /**
- * A typed adapter for the auto complete view, used to provide a layout, data binding and filter options for a single
- * data type. It holds an {@link AutoCompleteViewBinder} and a {@link TokenFilter}.
+ * A typed adapter for the auto complete view, used to provide a layout, data binding and filter options for a single data type.
+ * It holds an {@link AutoCompleteViewBinder} and a {@link TokenFilter}.
  *
  * @author Marco Salis
  */
 // @ThreadSafe
-class BaseTypeAdapterDelegate<M> extends BaseAdapter implements TypeAdapterDelegate<M>, AutoCompleteTypeAdapter<M> {
+class BaseTypeAdapterDelegate<M>
+        extends BaseAdapter
+        implements TypeAdapterDelegate<M>, AutoCompleteTypeAdapter<M> {
```

Comment:
I get a nullability lint warning here that we can suppress I'd say

Reply:
The only way to suppress it is to suppress any nullable problems in the whole class. A bit risky. We'll sort it out when we convert to Kotlin 👍